### PR TITLE
NFT Suggestion

### PR DESCRIPTION
**Pull request by MrRemiB**
Hi, thanks for your project and all your work

I want to suggest a new NFT, to reward the patience of budding farmers

BR and happy new year

### CODE REVIEW

File: src/dapp/types/crafting.ts [hunk 1]
```diff
@@ -274,6 +275,22 @@ export const recipes: Recipe[] = [
       "https://opensea.io/collection/sunflower-farmers-scarecrow",
     supply: 5000,
   },
+  {
+    name: "Pebble Tower",
+    abi : PebbleTower,
```
Comment (zinkkrysty) [inline src/dapp/types/crafting.ts:280]: you're missing an import for this one

File: src/dapp/components/farm/NFTs.tsx [hunk 2]
```diff
@@ -46,6 +47,11 @@ export const NFTs: React.FC<Props> = ({ inventory }) => {
           <img src={christmasTree} alt="christmasTree" />
         )}
       </div>
+      <div id="pebbletower">
+        {inventory["Pebbletower"] > 0 &&
```
Comment (zinkkrysty) [inline src/dapp/components/farm/NFTs.tsx:51]: shouldn't this be `Pebble Tower` ?

File: src/dapp/types/crafting.ts (hunk 1 above)
Comment (MrRemiB) [reply src/dapp/types/crafting.ts]: Ty for your review

File: src/dapp/components/farm/NFTs.tsx (hunk 2 above)
Comment (MrRemiB) [reply src/dapp/components/farm/NFTs.tsx]: Yes, ty, fixed

**Comment by adamhannigan** [comment]
Great to see the work you have done and welcome to the developer community! @MrRemiB 

The code looks good to me 🚀 

At it's current state the pixel art could do with a bit of a makeover. I would suggest heading over to the #designers channel in Discord and seeing if an artist can whip up something quickly for you.

**Comment by MrRemiB** [comment]
Thanks, good to know about the discord, i'll join to be a part of the community life :)
Just modified my pixel art to make it more detailed in 16x16

**Comment by MrRemiB** [comment]
Due to new community crafting feature... ABORT !

**Pull request closed without merging**